site/src/api/queries/organizations.ts

```diff
@@ -156,28 +158,91 @@ export const organizationPermissions = (organizationId: string | undefined) => {
             },
             action: "create",
           },
-          viewGroups: {
-            object: {
-              resource_type: "group",
-              organization_id: organizationId,
-            },
-            action: "read",
+        },
+      }),
+  };
+};
+
+/**
+ * Fetch permissions for all provided organizations.
+ *
+ * If organizations are undefined, return a disabled query.
+ */
+export const organizationsPermissions = (
+  organizations: Organization[] | undefined,
```

Comment:
since this only needs the `id`, can we just have it take a `string[]` instead and do a `.map` when we call it?

Reply:
Yup, works for me